var/spack/packages/cleverleaf/package.py

```diff
@@ -0,0 +1,14 @@
+from spack import *
+
+class Cleverleaf(Package):
+    homepage = "http://www.example.com"
+    url      = "http://www.example.com/cleverleaf-1.0.tar.gz"
+
+    version('develop', git='git@github.com:UK-MAC/CleverLeaf_ref.git', branch='develop')
```

Comment:
@davidbeckingsale: this fetch won't work if the user doesn't have ssh keys set up.  Can you update to use the http URL instead?